app/lib/sprites/SpriteBuilder.coffee

```diff
@@ -143,7 +143,19 @@ module.exports = class SpriteBuilder
 
```

Comment:
This file is the only change that could affect end users. But the change will only affect ThangTypes with ozaria set to true.